src/Jira/Api/Client/CurlClient.php

```diff
@@ -122,4 +123,27 @@ public function sendRequest(
 
         return $data;
     }
+	
+	/**
+      * If necessary, replace curl file @ string with a CURLFile object (for PHP 5.5 and up)
+      *
+      * @param string $fileString The string in @-format as it is used on PHP 5.4 and older.
+      * @return \CURLFile|string
+      */
+     protected function getCurlValue($fileString)
+     {
+         $theCurlValue = $fileString;
+         $regex = "|^@(.*);filename=(.*)$|";
```

Comment:
Is this regex for checking if given value isn't already in correct format that is created by `curl_file_create` function?